src/views/v2/list/event.php

```diff
@@ -19,7 +19,7 @@
 $container_classes = [ 'tribe-common-g-row', 'tribe-events-calendar-list__event-row' ];
 $container_classes['tribe-events-calendar-list__event-row--featured'] = $event->featured;
 
-$event_classes = get_post_class( [ 'tribe-events-calendar-list__event', 'tribe-common-g-row', 'tribe-common-g-row--gutters' ], $event->ID );
+$event_classes = tribe_get_post_class( [ 'tribe-events-calendar-list__event', 'tribe-common-g-row', 'tribe-common-g-row--gutters' ], $event->ID );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: e28191401aa05e13d077853ee88db6b6</sub>

Reply:
Ignoring.